tests/test_coverage.py

```diff
@@ -0,0 +1,182 @@
+# The MIT License
+#
+# Copyright (c) 2024 University of Strathclyde
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""Tests for the pyani_plus/db_orm.py module.
+
+These tests are intended to be run from the repository root using:
+
+pytest -v
+"""
+
+import tempfile
+from collections.abc import Callable
+from pathlib import Path
+
+import pytest
+
+from pyani_plus import db_orm, public_cli
+
+
+def do_comparison(
+    capsys: pytest.CaptureFixture[str],
+    fasta_dir: Path,
+    method: Callable,
+    **kwargs: float,
```

Comment:
Strangely although ``**kwargs`` refers to an optional dictionary of keyword arguments, for typing only the possible values are described. In our case this is currently ``int | float`` which is recorded as ``float``.